sitecore-packages.json

```diff
@@ -0,0 +1,311 @@
+{
+  "sitecore-9.1.1-xp0": {
```

Comment:
I notice you a Package "id" here in the json but the build.json files are using the file name. Wouldn't it be better to have the build.json file also use the package ids?

Reply:
So I retained structure of personal use for two reasons:

1. I can flex this format to be useful under this circmstance by transposing to a hashtable
2. I'm thinking it may be nice to leverage tokens over filenames in the future (have "sources" list `sc-9.2` over `Sitecore 9.2 rev. XXXXXXX.zip`.

I'm open to either though.

Reply:
I would recommend moving to tokens now if it's not too much work. In speaking with @bplasmeijer  and @pbering it seemed like an accepted approach.

Reply:
Yeah that was what we agreed on, to use the tokens in build.json to lookup in sitecore-packages.json. Is that something you can do in the PR?